blogposts/2020/release-3.24.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
@Joelkw this post is currently filed under /2020/. It looks like a new /2021/ folder needs to be created.